app/controllers/patient_admin/after_signup_controller.rb

```diff
@@ -0,0 +1,31 @@
+class PatientAdmin::AfterSignupController < PatientAdmin::ApplicationController
+  include Wicked::Wizard 
+ 
+  steps :personal
+
+  layout 'patient_auth'
+
+  def show
+    @personal = current_user.patient_personal
+    render_wizard
+  end
+
+  def update 
```

Comment:
A lot of code duplication, is it possible to inherit it from some base controller with list of the default actions or store shred methods in the concerns?